nekton/dcm2nii.py

```diff
@@ -61,27 +66,89 @@ def check_slice_thickness_variable(all_dcm_paths: List[str]) -> bool:
         )
         return False if len(all_slice_thickness) == 1 else True
 
-    def _run_conv_variable(self, all_dcm_paths: List[str], name: str) -> str:
+    def _run_conv_variable(self, dicom_directory: str) -> List[str]:
```

Comment:
You can use `Path` instead of str

Reply:
will do that for all the places where string paths are expected.